### PR TITLE
Conu test

### PR DESCRIPTION
**Pull request by phracek**
This Pull Request adds a test for memcached by conu.
For more information about conu, look here: https://github.com/fedora-modularity/conu/
RTD: http://conu.readthedocs.io/en/latest/

### CODE REVIEW

File: tests/memcached_conu.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+
+def test_conu():
+    """
+    Function tests memcached container with conu
+    """
+    backend = DockerBackend(logging_level=logging.DEBUG)
+    i = backend.ImageClass("docker.io/modularitycontainers/memcached")
+    i.pull()
+    i.get_metadata()
```
Comment (TomasTomecek) [inline tests/memcached_conu.py]: this doesn't do anything

File: tests/memcached_conu.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    rb = conu.DockerRunBuilder(command=["/files/memcached.sh"])
+    rb.build()
```
Comment (TomasTomecek) [inline tests/memcached_conu.py]: neither this

File: tests/memcached_conu.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    c = i.run_via_binary(rb)
+    assert(c.is_running() == True)
```
Comment (TomasTomecek) [inline tests/memcached_conu.py]: overly complicated, it's enough to just

```
assert c.is_running()
```

File: tests/memcached_conu.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    c.wait_for_port(11211)
+    session = pexpect.spawn("telnet %s 11211 " % c.get_IPv4s()[0])
+    session.sendline('set Test 0 100 10')
+    session.sendline('JournalDev')
+    assert (session.expect('STORED') == 0)
```
Comment (TomasTomecek) [inline tests/memcached_conu.py]: see above

File: tests/memcached_conu.py [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    backend = DockerBackend(logging_level=logging.DEBUG)
+    i = backend.ImageClass("docker.io/modularitycontainers/memcached")
+    i.pull()
+    i.get_metadata()
+    rb = conu.DockerRunBuilder(command=["/files/memcached.sh"])
+    rb.build()
+    c = i.run_via_binary(rb)
+    assert(c.is_running() == True)
+    c.wait_for_port(11211)
+    session = pexpect.spawn("telnet %s 11211 " % c.get_IPv4s()[0])
```
Comment (TomasTomecek) [inline tests/memcached_conu.py]: this is nice, I would love if we had this as an official example

**Review by TomasTomecek — CHANGES_REQUESTED** [review]
a couple suggestions, otherwise lgtm

File: tests/memcached_conu.py (hunk 5 above)
Comment (phracek) [reply tests/memcached_conu.py]: Go ahead. Feel free to share it.

File: tests/memcached_conu.py (hunk 4 above)
Comment (phracek) [reply tests/memcached_conu.py]: I remove brakes, but `assert session.expect('STORED')` finish with traceback like:
```
>       assert session.expect('STORED')
E       AssertionError: assert 0
E        +  where 0 = <bound method spawn.expect of <pexpect.pty_spawn.spawn object at 0x7f83cc3bfe10>>('STORED')
E        +    where <bound method spawn.expect of <pexpect.pty_spawn.spawn object at 0x7f83cc3bfe10>> = <pexpect.pty_spawn.spawn object at 0x7f83cc3bfe10>.expect

```

Comment (phracek) [reply tests/memcached_conu.py]: See `/usr/lib/python2.7/site-packages/pexpect/spawnbase.py` and row 276 as an example in function pexpect.

**Review by TomasTomecek — APPROVED** [review]

**Pull request merged**